argon2/src/lib.rs

```diff
@@ -68,7 +68,8 @@
     html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
     html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
 )]
-#![forbid(unsafe_code)]
+#![cfg_attr(not(feature = "parallel"), forbid(unsafe_code))]
+#![cfg_attr(feature = "parallel", deny(unsafe_code))]
```

Comment:
I think it's fine to just change this to:

```suggestion
#![deny(unsafe_code)]
```